kernel-shared/uuid-tree.h

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
Also please add uuid-tree.h to README.md

Reply:
Added to the `kernel-shared/README.h`.

And I find the automatic `outdated` tag very useful to locate missing fixes.
Although in this particular case, there is not `outdated` tag since the change is done to another file.